admin_interface/templates/admin/change_form.html

```diff
@@ -0,0 +1,68 @@
+{% extends "admin/change_form.html" %}
+{% load static admin_interface_tags %}
+
+
+{% block field_sets %}
+
+{% get_admin_interface_setting "show_fieldsets_as_tabs" as show_fieldsets_as_tabs %}
+{% get_admin_interface_setting "show_inlines_as_tabs" as show_inlines_as_tabs %}
+
+
```

Comment:
If both `show_fieldsets_as_tabs` and `show_inlines_as_tabs` are False, just call `{{ block.super }}`, in this way if there is any bug with this feature the users can just deactivate it to use the initial change form template.

Reply:
Updated